roles/kubernetes-apps/policy_controller/meta/main.yml

```diff
@@ -4,13 +4,13 @@ dependencies:
     when:
       - kube_network_plugin == 'calico'
       - enable_network_policy
-      - calico_datastore != "kdd"
+      - calico_datastore != "kdd" or calico_policy_version is version('v3.6.0', '>=')
```

Comment:
The default version of `calico_policy_version` comes from `calico_version` and the minimum supported version is 3.19.4.

So this additional condition (`calico_policy_version is version('v3.6.0', '>=')`) is always true.
I think we need to remove the line 7 completely instead of reverting the condition.

Reply:
I agree on this. That's what I was thinking on the beginning, but I wasn't sure. You're right: https://github.com/kubernetes-sigs/kubespray/blob/master/roles/kubespray-defaults/vars/main.yml#L7

Reply:
Thanks for updating @wayfrro